subsys/bluetooth/mesh/access.h

```diff
@@ -52,3 +54,26 @@ void bt_mesh_model_bind_store(struct bt_mesh_model *mod);
 void bt_mesh_model_sub_store(struct bt_mesh_model *mod);
 void bt_mesh_model_pub_store(struct bt_mesh_model *mod);
 void bt_mesh_model_settings_commit(void);
+
+/** @brief Register a callback function hook for mesh model messages.
+ *
+ * Register a callback function to act as a hook for recieving mesh model layer messages
+ * directly to the application without having instantiated the relevant models.
+ *
+ * @param cb A pointer to the callback function.
+ */
+void bt_mesh_msg_cb_set(void (*cb)(uint32_t opcode, struct bt_mesh_msg_ctx *ctx,
+			 struct net_buf_simple *buf));
+
+/** @brief Send a mesh model message.
+ *
+ * Send a mesh model layer message out into the mesh network without having instantiated
+ * the relevant mesh models.
+ *
+ * @param ctx The Bluetooth mesh message context.
+ * @param buf The message payload.
+ *
+ * @return 0 on success or negative error code on failure.
+ */
+int bt_mesh_msg_send(struct bt_mesh_msg_ctx *ctx, struct net_buf_simple *buf, uint16_t src_addr,
+		const struct bt_mesh_send_cb *cb, void *cb_data);
```

Comment:
Minor: the split line doesn't look like it's correctly aligned

Reply:
Ah yes, I'll correct this.